<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProntoSoccorso</class>
 <widget class="QWidget" name="ProntoSoccorso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 235, 239);</string>
  </property>
  <widget class="QLabel" name="InformazioniProfilo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>581</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);
color: rgb(230, 235, 239);</string>
   </property>
   <property name="text">
    <string>Lettera di dimissione</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonRegistra">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>290</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(0, 57, 97,0);
	color: rgb(230, 235, 239);
color: rgb(230, 235, 239);
border-radius: 10px;
}
QPushButton:hover {
background-color: rgba(40, 40, 40,0.5);
    }</string>
   </property>
   <property name="text">
    <string>Registra</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>450</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>E:/UI UFFICIALI/MATERIALE/Back.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>155</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(230, 235, 239,0.6);
background-color: rgba(0, 57, 97,0);</string>
   </property>
   <property name="text">
    <string>Ciclo di cure domiciliare</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditCicloDiCure">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>290</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border: 1px solid rgba(230, 235, 239,0.6);
border-radius: 6px;
padding: 5px;
}
                            
QTextEdit:focus {
background-color: rgba(40, 40, 40,0.7);
}





QScrollBar:vertical {
background:rgba(0,0,0,0);
width: 5px;
margin: 0px 0 0px 0;


}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
background-color: rgb(40, 40, 40);
border-radius: 2.2px;
}

QScrollBar::handle:vertical {
background-color: rgba(230, 235, 239,0.5);
min-height: 20px;
border-radius: 2px;
}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Inserisci qui</string>
   </property>
  </widget>
  <widget class="QLabel" name="LogoLettarDiDimissione">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>129</y>
     <width>192</width>
     <height>195</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>E:/UI UFFICIALI/MATERIALE/IconaLetteraDiDimissione.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(230, 235, 239,0.6);
background-color: rgba(0, 57, 97,0);</string>
   </property>
   <property name="text">
    <string>Dati Paziente</string>
   </property>
  </widget>
  <widget class="QLabel" name="Nome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>105</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);

border-radius: 6px;
   

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Cognome">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>105</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);

border-radius: 6px;
   

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Sheda_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>95</y>
     <width>290</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);
border: 1px solid rgba(230, 235, 239,0.6);
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Sheda_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>315</y>
     <width>290</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);
border: 1px solid rgba(230, 235, 239,0.6);
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);

border-radius: 6px;
   

</string>
   </property>
   <property name="text">
    <string>Data dimissione:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(230, 235, 239,0.6);
background-color: rgba(0, 57, 97,0);</string>
   </property>
   <property name="text">
    <string>Dimissione</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Data">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);

border-radius: 6px;
   

</string>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>Back</zorder>
  <zorder>Sheda_2</zorder>
  <zorder>InformazioniProfilo</zorder>
  <zorder>ButtonRegistra</zorder>
  <zorder>label_5</zorder>
  <zorder>textEditCicloDiCure</zorder>
  <zorder>LogoLettarDiDimissione</zorder>
  <zorder>label_8</zorder>
  <zorder>Nome</zorder>
  <zorder>Cognome</zorder>
  <zorder>Sheda_3</zorder>
  <zorder>label_6</zorder>
  <zorder>label_9</zorder>
  <zorder>lineEdit_Data</zorder>
 </widget>
 <resources>
  <include location="E:/UI UFFICIALI/asa.qrc"/>
 </resources>
 <connections/>
</ui>
